src/util.ts

```diff
@@ -1,7 +1,8 @@
-import * as stableStringify from 'json-stable-stringify';
+import * as stableStringify_ from 'json-stable-stringify';
 import {isArray, isNumber, isString, splitAccessPath, stringValue} from 'vega-util';
 import {isLogicalAnd, isLogicalNot, isLogicalOr, LogicalOperand} from './logical';
 
+const stableStringify = stableStringify_['default'] || stableStringify_;
```

Comment:
Why do we need to do this?  Shouldn't it be just one of these two?

Reply:
This is just to be safe. Rollup can change how modules are exposed depending on what plugin you use. This always works.